var/spack/repos/builtin/packages/py-prometheus-client/package.py

```diff
@@ -13,6 +13,10 @@ class PyPrometheusClient(PythonPackage):
     url      = "https://pypi.io/packages/source/p/prometheus_client/prometheus_client-0.7.1.tar.gz"
 
     version('0.7.1', sha256='71cd24a2b3eb335cb800c7159f423df1bd4dcd5171b234be15e3f31ec9f622da')
+    version('0.7.0', sha256='ee0c90350595e4a9f36591f291e6f9933246ea67d7cd7d1d6139a9781b14eaae')
     version('0.5.0', sha256='e8c11ff5ca53de6c3d91e1510500611cafd1d247a937ec6c588a0a7cc3bef93c')
 
+    variant('twisted', default=True, description='Expose metrics as a twisted resource')
```

Comment:
I usually set variants that add extra dependencies to False by default. Do you have any strong opinions on this?

Reply:
Nope.